core/tx_pool.go

```diff
@@ -681,6 +682,14 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (replaced bool, err e
 	// the sender is marked as local previously, treat it as the local transaction.
 	isLocal := local || pool.locals.containsTx(tx)
 
+	if pool.chainconfig.UsingScroll {
```

Comment:
For this part, transactions that don’t have enough fees will be dismissed directly by the transaction pool. I think it could be hard for the user to know if their transactions are rejected. This event is hard to directly notify users unless they are monitoring all the tx pools?